packages/menu/src/Menu.ts

```diff
@@ -262,6 +262,12 @@ export class Menu extends SpectrumElement {
         }
         this.observer.observe(this, { childList: true, subtree: true });
         this.updateComplete.then(() => this.prepItems());
+        const selectedItem = this.querySelector('[selected]');
```

Comment:
Seems like this should be good with multi-select too, as it should scroll the first selected item into view.